appveyor.yml

```diff
@@ -2,8 +2,6 @@ version: 1.0.{build}
 
 environment:
   matrix:
-  - TOXENV: cleanup
-  - TOXENV: docs
```

Comment:
Sure, we shouldn't be checking for `docs`?

Reply:
This is what the tox -e ..,docs,... line does below.

Reply:
ahh, of course. thanks for the clarification! :-)